servers/quarkus-common/src/main/java/org/projectnessie/quarkus/config/CatalogServiceConfig.java

```diff
@@ -22,6 +22,14 @@
 
 @ConfigMapping(prefix = "nessie.catalog.service")
 public interface CatalogServiceConfig {
+  /**
+   * Interval after which a request is retried when storage I/O responds with some "retry later"
+   * response.
+   */
+  @WithName("throttled-retry-after")
+  @WithDefault("PT10S")
+  Duration retryAfterThrottled();
```

Comment:
If this is moved to `CatalogConfig` (and `QuarkusCatalogConfig`), `EntitySnapshotTaskBehavior` doesn't need to be produced in `CatalogProducers` can can stay package-private.

Reply:
Would you like this config to be moved to `CatalogConfig`?

My impressions is that `CatalogConfig` contains user-level settings, but `throttled-retry-after` is more like an admin/service level setting (which is why I put it into `CatalogServiceConfig`). WDYT?

Reply:
`CatalogServiceConfig` is rather Quarkus specific, while `CatalogConfig` is "generic".